frontend/src/components/common/form/Check.scss

```diff
@@ -7,3 +8,10 @@
     padding-right: 5px;
   }
 }
+.required.form-group {
```

Comment:
Sorry, another distraction. When we removed all of the required asterixes we forgot to add something to the checkboxes to indicate required.

I played around a bit and came up with this (note that styling the checkbox box itself in css is actually extremely annoying):

![image](https://user-images.githubusercontent.com/41091511/101855579-844fac80-3b18-11eb-8269-86d66dbfc7e1.png)

Reply:
Looks good to me, seems to fit the rest of our required styling as well